### PR TITLE
Fixed #25959 -- Updated documentation of django logger.

### PR DESCRIPTION
**Pull request by varunsharma**
Changed the default logging example in documentation from django.request to django.

### CODE REVIEW

**Comment by timgraham** [comment]
I'd rather not make the example more complicated and add the email stuff. What was your reasoning for that?

File: docs/topics/logging.txt [hunk 1]
```diff
@@ -241,25 +241,37 @@ information about logging configuration dictionaries. However, to give
 you a taste of what is possible, here are several examples.
 
 First, here's a simple configuration which writes all request logging from the
```
Comment (timgraham) [inline docs/topics/logging.txt]: can remove "request" here

File: docs/topics/logging.txt [hunk 2]
```diff
@@ -241,25 +241,37 @@ information about logging configuration dictionaries. However, to give
...
-:ref:`django-request-logger` logger to a local file::
+:ref:`django-logger` logger to console if ``DEBUG=True`` and sends email to admins if ``DEBUG=False``::
```
Comment (timgraham) [inline docs/topics/logging.txt]: Paragraphs should be wrapped at 79 characters

File: docs/topics/logging.txt [hunk 3]
```diff
@@ -241,25 +241,37 @@ information about logging configuration dictionaries. However, to give
...
         'loggers': {
-            'django.request': {
-                'handlers': ['file'],
-                'level': 'DEBUG',
-                'propagate': True,
+            'django': {
+                'handlers': ['console', 'mail_admins'],
+                'level': 'INFO',
             },
-        },
```
Comment (timgraham) [inline docs/topics/logging.txt]: We prefer including a trailing comma so that if the dict is extended later, we don't have to modify that line again.

**Comment by varunsharma** [comment]
Hi Tim,
I just tried to update the example as per changes done in commit 8efea1b. I was also looking to leave out some complicated settings values like filters but when I tested it on my local environment, it was giving me ValueError in config.py. So, I added the entire default dict in docs. Let me know if there is a better way to do it. Also, I've improved the PR as per your inline comments. Thanks.

**Comment by timgraham** [comment]
Any problem with this?
```diff
diff --git a/docs/topics/logging.txt b/docs/topics/logging.txt
index d64db31..99fdad0 100644
--- a/docs/topics/logging.txt
+++ b/docs/topics/logging.txt
@@ -240,8 +240,8 @@ The full documentation for `dictConfig format`_ is the best source of
 information about logging configuration dictionaries. However, to give
 you a taste of what is possible, here are several examples.
 
-First, here's a simple configuration which writes all request logging from the
-:ref:`django-request-logger` logger to a local file::
+First, here's a simple configuration which writes all logging from the
+:ref:`django-logger` logger to a local file::
 
     LOGGING = {
         'version': 1,
@@ -254,7 +254,7 @@ First, here's a simple configuration which writes all request logging from the
             },
         },
         'loggers': {
-            'django.request': {
+            'django': {
                 'handlers': ['file'],
                 'level': 'DEBUG',
                 'propagate': True,
@@ -459,6 +459,8 @@ Loggers
 
 Django provides several built-in loggers.
 
+.. _django-logger:
+
 ``django``
 ~~~~~~~~~~
 ```

**Comment by varunsharma** [comment]
Sorry, I didn't know that old file handlers are still compatible with this.
Creating another PR as this one got closed automatically for some strange reason after I updated my branch with master.

**Pull request closed without merging**